### PR TITLE
blog: Add tips and tricks article

### PR DESCRIPTION
**Pull request by kingthorin**
I could really use help on the image, I am NOT an artist.

<details>
<summary>Old Blog Screenshot</summary>

image

</details>

image

### CODE REVIEW

**Comment by kingthorin** [comment]
I've dated it for a week from now 🤷‍♂️ it can be adjusted as needed, there's no particular reason one way or another.

I also know it's short and I've been talking about it for a while, let's just say I've had some motivation issues 😟

File: site/content/blog/2023-08-08-community-tips-and-tricks/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+date: "2023-08-08"
+authors:
+    - thorin
+---
+
+![](images/zapbot.TandT.png)
+
+# Community - Tips and Tricks
+
+We've established a [community area](https://github.com/zaproxy/community-scripts/tree/main/other/tips) to publish people's Tips and Tricks for using ZAP and it's add-ons. Please feel free to open PRs with your best ideas.
```
Comment (thc202) [inline site/content/blog/2023-08-08-community-tips-and-tricks/index.md:18]: `its`

**Comment by thc202** [comment]
The content looks good to me, it does not need to be always a wall of text :) For me, any day once we are all happy with it.

I can try tweak the image but you'll have to tell how do you want it.

**Comment by kingthorin** [comment]
I dunno I just didn't feel like I'd done a good job on the light bulb or slate thing...

**Comment by kingthorin** [comment]
@psiinon thoughts?

**Comment by ricekot** [comment]
Not sure if this is any good, but I used the monthly blog post ZAPbot template and put it on a lightning GIF.
I need to figure out how to move ZAPBot's arms around :smile: 

zap-tips-and-tricks

**Comment by kingthorin** [comment]
Yeah, that's fun. Thank you!!

**Comment by kingthorin** [comment]
Done & done.

File: site/content/blog/2023-08-25-community-tips-and-tricks/index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Community - Tips and Tricks"
+summary: >
+  News about a community area to contribute ZAP usage tips and tricks.
+images:
+- https://www.zaproxy.org/blog/2023-08-08-community-tips-and-tricks/images/zapbot.TandT.gif
```
Comment (kingthorin) [inline site/content/blog/2023-08-25-community-tips-and-tricks/index.md:6]: Tweak date

**Comment by kingthorin** [comment]
Please review, there is one minor thing I should tweak in the morning but otherwise I believe this is good to go.

**Comment by kingthorin** [comment]
Fixed

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

File: site/content/blog/2023-08-25-community-tips-and-tricks/index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: "Community - Tips and Tricks"
+summary: >
+  News about a community area to contribute ZAP usage tips and tricks.
+images:
+- https://www.zaproxy.org/blog/2023-08-25-community-tips-and-tricks/images/zapbot.TandT.gif
+addSocialPreview: true
+type: post
+date: "2023-08-25"
```
Comment (psiinon) [inline site/content/blog/2023-08-25-community-tips-and-tricks/index.md:9]: No tags?
We usually include at least "blog"

File: site/content/blog/2023-08-25-community-tips-and-tricks/index.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+authors:
+    - thorin
+---
+
+![](images/zapbot.TandT.gif)
+
+# Community - Tips and Tricks
+
+We've established a [community area](https://github.com/zaproxy/community-scripts/tree/main/other/tips) to publish people's Tips and Tricks for using ZAP and its add-ons. Please feel free to open PRs with your best ideas.
```
Comment (psiinon) [inline site/content/blog/2023-08-25-community-tips-and-tricks/index.md:18]: I like it but it looks very sparce at the moment :/
Are there any other tips-n-tricks we could include?
I dont have huge expectations for the community to share loads more with us ..

Comment (kingthorin) [reply site/content/blog/2023-08-25-community-tips-and-tricks/index.md:18]: I don't have anything else ready off the top of my head. But it has to start somewhere.

File: site/content/blog/2023-08-25-community-tips-and-tricks/index.md (hunk 3 above)
Comment (kingthorin) [reply site/content/blog/2023-08-25-community-tips-and-tricks/index.md:9]: Hmmm not sure how I missed that, I'll get some added.

**Comment by kingthorin** [comment]
Now with tags.

**Review by psiinon — APPROVED** [review]

**Pull request merged**